src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMA = ",";
```

Comment:
It is obvious that it is coma. But it's not a good name for constant here.
Constant name should provide information about the purpose of this variable.
It could be named like: DELIMITER